ibis/backends/duckdb/registry.py

```diff
@@ -177,6 +178,25 @@ def _string_agg(t, expr):
     return agg
 
 
+def _struct_column(t, expr):
+    op = expr.op()
+    return sa.func.struct_pack(
+        *map(
+            sa.text,
+            map(
+                "{} := {}".format,
+                op.names,
+                map(
+                    methodcaller(
+                        "compile", compile_kwargs=dict(literal_binds=True)
+                    ),
+                    map(t.translate, op.values),
+                ),
+            ),
+        )
```

Comment:
Small nit: I find a single generator expression to be much more readable than nested map calls. Untested (and unlinted), but I believe the following is equivalent:

```suggestion
        *(
            sa.text(f"{name} := {t.translate(value).compile(literal_binds=True)}")
            for name, value in zip(op.names, op.values)
        )
```

Reply:
Yeah, that's better. Thanks!